src/python/pants/option/config_test.py

```diff
@@ -77,35 +78,41 @@ class ConfigFile:
         "embed": "%(path)s::%(name)s",
         "disclaimer": "Let it be known\nthat.",
     },
-    expanded_default_values={
-        "name": "foo",
-        "answer": 42,
-        "scale": 1.2,
-        "path": "/a/b/42",
-        "embed": "/a/b/42::foo",
-        "disclaimer": "Let it be known\nthat.",
-    },
     expected_options={
-        "a": {"list": "['1', '2', '3', '42']", "list2": "+[7, 8, 9]", "list3": "-['x', 'y', 'z']"},
-        "b": {"preempt": "True"},
+        "DEFAULT": {
+            "name": ["foo"],
+            "answer": ["42"],
+            "scale": ["1.2"],
+            "path": ["/a/b/42"],
+            "embed": ["/a/b/42::foo"],
+            "disclaimer": ["Let it be known\nthat."],
+        },
+        "a": {
```

Comment:
The diffs here are:
1. Adding the DEFAULT values
2. Wrapping all the values in lists, so that the _compare function can always expect lists, even if they are singletons in the case of a single config file.